syncd/syncd_flex_counter.cpp

```diff
@@ -74,11 +157,12 @@ void FlexCounter::setQueueCounterList(
 void FlexCounter::setQueueAttrList(
         _In_ sai_object_id_t queueVid,
         _In_ sai_object_id_t queueId,
+        _In_ uint32_t pollInterval,
         _In_ const std::vector<sai_queue_attr_t> &attrIds)
 {
     SWSS_LOG_ENTER();
 
-    FlexCounter &fc = getInstance();
+    FlexCounter &fc = getInstance(pollInterval);
```

Comment:
>getInstance [](start = 22, length = 11)

remove instance?

Reply:
Why do we need to remove the instance after creating? 
The port/queue list will be added to the instance. 
At least, we don't know if instance is just created or it has been already created.